File: hello-world/python/send.py
#!/usr/bin/env python
import pika

# Create connection to rabbitmq server
connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
channel = connection.channel()

# Create queue unless rabbit will drop our message
channel.queue_declare(queue='hello')

# Use default exchanger (no name) for first hello world
# routing_key = queue name
channel.basic_publish(
    exchange='',
    routing_key='hello',
    body='Hello World!'
)

print(" [x] Sent 'Hello World!'")

# Close connection
connection.close()

File: hello-world/ruby/send.rb
#!/usr/bin/env ruby
require 'bunny'

connection = Bunny.new
connection.start

channel = connection.create_channel

# create queue
queue = channel.queue('hello')

# publish message
channel.default_exchange.publish('Hello World!', routing_key: queue.name)

puts " [x] Sent 'Hello World!'"

connection.close